Geometry/GEMGeometryBuilder/plugins/BuildFile.xml

```diff
@@ -6,29 +6,37 @@
 <use name="FWCore/ParameterSet"/>
 <use name="Geometry/MuonNumbering"/>
 <use name="Geometry/Records"/>
+
 <library name="GEMGeometryPlugins" file="GEMGeometryESModule.cc">
-  <use name="DetectorDescription/Core"/>
-  <use name="DetectorDescription/DDCMS"/>
-  <use name="Geometry/GEMGeometry"/>
-  <use name="dd4hep"/>
-  <lib name="Geom"/>
-  <use name="Geometry/GEMGeometryBuilder"/>
-  <flags EDM_PLUGIN="1"/>
+<use name="DetectorDescription/Core"/>
```

Comment:
it looks like you undid the buildfile changes from the recent xml formatting pass, please revert

Reply:
@kpedro88 : now "diff new_BuildFile.xml old_BuildFile.xml" is related only to the ME0 Validation part:
< 
< <library name="ME0GeometryValidationPlugins" file="ME0GeometryValidate.cc">
<   <use name="Fireworks/Core"/>
<   <use name="Geometry/GEMGeometry"/>
<   <use name="rootgeom"/>
<   <flags EDM_PLUGIN="1"/>
< </library>

Reply:
@kpedro88 : I restored the old format and I added only the ME0 validation part